assemble/src/main/scripts/create-jshell.sh

```diff
@@ -40,19 +40,29 @@ function addClientBuild() {
 cat <<EOF
   URL clientPropUrl = 
     AccumuloClient.class.getClassLoader().getResource("accumulo-client.properties");
-  String accumuloProp; AccumuloClient client = null;
-  
+  AccumuloClient client = null;
+      
   // Does Accumulo properties exists?
   if (clientPropUrl != null) {
   
-    // Build Accumulo Client
-    accumuloProp = clientPropUrl.getFile();
-    System.out.println("Building Accumulo client using properties file below: \n" + accumuloProp);
-    client = Accumulo.newClient().from(accumuloProp).build();
-    System.out.println("Use " + "\"client\"" + " to interact with Accumulo \n");
-  } else {
-      System.out.println("accumulo-client.properties was not found on the classpath \n");
-  }
+    // Load in Accumulo properties
+    try (InputStream in = clientPropUrl.openStream()) {
+      Properties prop = new Properties(); 
+      prop.load(in);
+      
+      // Build Accumulo Client
+      System.out.println("Building Accumulo client using '" + clientPropUrl + "'\n");
+      prop.put("instance.name", prop.getProperty("instance.name"));
+      prop.put("instance.zookeepers", prop.getProperty("instance.zookeepers"));
+      prop.put("auth.type", prop.getProperty("auth.type"));
+      prop.put("auth.principal", prop.getProperty("auth.principal"));
+      prop.put("auth.token", prop.getProperty("auth.token"));
```

Comment:
Aren't these redundant with the call to load?

Reply:
Yes you are right. At the time I was using the Java Properties object method to test out the input stream. I wanted to get a second opinion on the input stream method first before simplifying the implementation. Will push a new commit so that it uses the [builder method](https://accumulo.apache.org/docs/2.x/getting-started/clients)